tlatools/org.lamport.tlatools/.classpath

```diff
@@ -26,11 +26,15 @@
 	<classpathentry kind="lib" path="lib/jline/jline-builtins-3.25.0.jar"/>
 	<classpathentry kind="lib" path="lib/jline/jline-reader-3.25.0.jar"/>
 	<classpathentry kind="lib" path="lib/jline/jline-terminal-3.25.0.jar"/>
-	<classpathentry kind="lib" path="lib/jpf.jar"/>
 	<classpathentry kind="lib" path="lib/jmh/jmh-core-1.21.jar"/>
 	<classpathentry kind="lib" path="lib/jmh/commons-math3-3.2.jar"/>
 	<classpathentry kind="lib" path="lib/gson/gson-2.8.6.jar"/>
 	<classpathentry kind="lib" path="lib/lsp/org.eclipse.lsp4j.debug_0.21.1.v20230829-0012.jar"/>
 	<classpathentry kind="lib" path="lib/lsp/org.eclipse.lsp4j.jsonrpc_0.21.1.v20230829-0012.jar"/>
+	<classpathentry kind="lib" path="lib/junit-4.12.jar"/>
+	<classpathentry kind="lib" path="lib/jpf.jar"/>
+	<classpathentry kind="lib" path="lib/easymock-3.3.1.jar"/>
```

Comment:
Adding these libraries to .classpath will cause the to be part of the tlatools jar that is part of the Toolbox.

Reply:
How then should the project be set up so unit tests can be run from eclipse without having to load in all the toolbox stuff?

Reply:
Reverted these changes in this branch.